# Index-error
 am trying to run spladder_test.py script for Differential expression as following :  python spladder-development/python/spladder_test.py -o ./spladder_output -a Col-0_6hpi-rep1_trimmed.sort.bam,Col-0_6hpi-rep2_trimmed.sort.bam,Col-0_6hpi-rep3_trimmed.sort.bam -b Col-0_48hpi-rep1_trimmed.sort.bam,Col-0_48hpi-rep2_trimmed.sort.bam,Col-0_48hpi-rep3_trimmed.sort.bam -n 150 -c 3 -D y --labelA=6hpi --labelB=48hpi -t exon_skip,intron_retension,alt_3prime,alt_5prime,mutex_exons,mult_exon_skip -v y  I get index error :  File "spladder-development/python/spladder_test.py", line 805, in main gene_counts, gene_strains, gene_ids = get_gene_expression(CFG, fn_out=CFG['fname_exp_hdf5'], strain_subset=condition_strains) File "spladder-development/python/spladder_test.py", line 176, in get_gene_expression gene_counts[gidx, :] = IN['segments'][seg_idx, :][strain_idx] * seg_lens[seg_idx] / CFG['read_length'] IndexError: index 6 is out of bounds for axis 0 with size 1  What does this error means and how can I solve this please?
